src/tools/linkchecker/main.rs

```diff
@@ -138,6 +138,11 @@ fn walk(cache: &mut Cache, root: &Path, dir: &Path, errors: &mut bool) {
 }
 
 fn is_exception(file: &Path, link: &str) -> bool {
+    // FIXME: These links are not resolved in trait impls due to #73829.
+    if link.ends_with("Clone::clone") || link.ends_with("Iterator::try_fold") {
+        return true;
+    }
```

Comment:
Can you put these in `LINKCHECK_EXCEPTIONS` instead so you don't accidentally allow more broken links than you meant to?

cc @Manishearth who wrote this originally

Reply:
No, because there are hundreds of broken links.